### PR TITLE
Gcw 2370 improve offer search

### PR DESCRIPTION
**Pull request by steveyken**
Added the following improvements to the offer search:

- Search for chinese package name
- Search for gogovan driver name
- Search for gogovan driver mobile
- Search for gogovan driver license
- Don't return duplicate results

### CODE REVIEW

File: spec/models/concerns/push_updates_for_offer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# require 'rails_helper'
```
Comment (patrixr) [inline spec/models/concerns/push_updates_for_offer_spec.rb]: This looks odd. Was it a debug that got pushed by mistake ? : )

File: app/models/concerns/offer_search.rb [hunk 2]
```diff
@@ -25,7 +26,10 @@ module OfferSearch
           joins("LEFT OUTER JOIN items ON offers.id = items.offer_id").
           joins("LEFT OUTER JOIN messages ON offers.id = messages.offer_id OR items.id = messages.item_id").
           joins("LEFT OUTER JOIN packages ON packages.item_id = items.id").
-          joins("LEFT OUTER JOIN package_types ON package_types.id = packages.package_type_id")
+          joins("LEFT OUTER JOIN package_types ON package_types.id = packages.package_type_id").
```
Comment (namrataukirde) [inline app/models/concerns/offer_search.rb]: Place the `.` on the next line, together with the method name.

From code climate.

File: spec/models/concerns/push_updates_for_offer_spec.rb (hunk 1 above)
Comment (steveyken) [reply spec/models/concerns/push_updates_for_offer_spec.rb]: oops... didn't mean to include this file!

File: app/models/concerns/offer_search.rb (hunk 2 above)
Comment (patrixr) [reply app/models/concerns/offer_search.rb]: don't do it @steveyken  code climate is evil 😆

**Pull request merged**